code/_onclick/hud/parallax.dm

```diff
@@ -133,7 +133,7 @@
 	if (C.parallax_animate_timer)
 		deltimer(C.parallax_animate_timer)
 	var/datum/callback/CB = CALLBACK(src, .proc/update_parallax_motionblur, C, animatedir, new_parallax_movedir, newtransform)
-	if(skip_windups)
+	if(skip_windups || isnull(shortesttimer))
 		CB.Invoke()
```

Comment:
??

Reply:
`shortesttimer` never gets assigned a value if we don't have any parallax layers (so if we have the parallax pref set to none). So if you pass it further down into the 
`C.parallax_animate_timer = addtimer(CB, min(shortesttimer, PARALLAX_LOOP_TIME), TIMER_CLIENT_TIME|TIMER_STOPPABLE)` line the timer becomes a 0 second timer which is just a runtime.